### PR TITLE
Adding module in event edits

### LINKED ISSUE #2098: Ticketing section not shown in edit-event form
**Describe the bug**

The ticketing section is not shown in edit-events form even though the Ticketing module has been enabled in the system

**To Reproduce**
1. Go to editing of any event
2. Scroll down to Organiser Information Tab (just above where ticketing should be)

**Expected behavior**

Ticket options should be there when module is activated

**Screenshots**

editingevent
moduleactivated

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] Ubuntu 16.04 LTS
 - Browser [e.g. chrome, safari] Chrome
 
**Additional context**

I am working on this

### PR DESCRIPTION
**Pull request by uds5501**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
After this PR, the newly added changes for `basic-details-step` can be reflected in the `edit-form` steps too which was earlier not handled

#### Changes proposed in this pull request:

- Included module model in `events.view.edit`

postfix

Fixes #2098

### CODE REVIEW

**Comment by uds5501** [comment]
@CosmicCoder96 Did so

File: app/routes/events/view/edit.js [hunk 1]
```diff
@@ -21,7 +21,8 @@ export default Route.extend(AuthenticatedRouteMixin, EventWizardMixin, {
       types : await this.store.query('event-type', {
         sort: 'name'
       }),
-      topics: await this.store.query('event-topic', {
+      module : await this.get('store').queryRecord('module', {}),
```
Comment (niranjan94) [inline app/routes/events/view/edit.js]: Use ES5 getters whenever possible. `this.store` instead of `this.get('store')`

Comment (uds5501) [reply app/routes/events/view/edit.js]: Sure

**Comment by uds5501** [comment]
@niranjan94 Squashed it

**Pull request merged**